src/components/ErrorView.js

```diff
@@ -45,7 +46,7 @@ const ErrorView = ({ error }) => {
                 return h(Fragment, [
                   json.message && div({ style: { marginBottom: '1rem' } }, [json.message]),
                   div({ style: { fontWeight: 600, marginBottom: '0.5rem' } }, ['Full error:']),
-                  div({ style: styles.jsonFrame }, [JSON.stringify(json, null, 2)]),
+                  div({ style: { ...styles.jsonFrame, backgroundColor: colors.light() } }, [JSON.stringify(json, null, 2)]),
```

Comment:
Is this backgroundColor a change or just a refactoring of where it comes from?

Reply:
and I guess, is there a reason for attaching it directly to the component instead of making it part of a style (or indeed, the `jsonFrame` style given the updates below too)?